projects/plugins/jetpack/extensions/blocks/ai-chat/block.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
```

Comment:
Since we support the current version of WP (6.3) minus 1, so currently WP 6.2, should we keep this to version 2, to be compatible with the previous version of WordPress?

Reply:
I suppose we can give it a try as v3 for now, and see what happens :)